deeds-dapp-webapp/src/main/webapp/vue-app/dapp/staticPage/components/page/productTour.vue

```diff
@@ -56,9 +55,8 @@
         <v-btn
           href="https://builders.meeds.io"
           target="_blank"
-          width="300px"
-          height="75px"
-          class="dark-grey-border-color dark-grey-color d-flex justify-center mx-5 mt-7" 
+          height="70px"
+          class="primary--text primary-border-color d-flex justify-center px-8 mx-5 mt-7" 
```

Comment:
could you test this instead please:

```suggestion
          color="primary"
          class="d-flex justify-center px-8 mx-5 mt-7" 
```

Reply:
done